pkg/awsutil/sts.go

```diff
@@ -123,8 +128,11 @@ var DefaultSTSClientProviderFunc STSClientProviderFunc = func(optnFns ...func(op
 }
 
 type RoleArnSession struct {
+	Name            string
 	RoleSessionName string
 	RoleArn         string
+	IsCustomerRole  bool
```

Comment:
What is the use case for this in the struct? Only for printing the debug log?

Reply:
Basically, to identify the customer role to apply the session policy ARN , otherwise we always have to check if `name=CustomerRoleArn` statment